components/display/index.rst

```diff
@@ -35,7 +35,10 @@ individually.
 
 So, first a few basics: When setting up a display platform in ESPHome there will be a configuration
 option called ``lambda:`` which will be called every time ESPHome wants to re-render the display.
-In there, you can write code like in any :ref:`lambda <config-lambda>` in ESPHome. Display
+In each cycle, the display is automatically cleared before the lambda is executed. For some
+displays (e.g., :doc:`ili9341`) you can disable this behavior by setting
+``auto_clear_enabled: false``.
```

Comment:
The "some displays" part isn't correct anymore since this was added to `FULL_DISPLAY_SCHEMA` (note that this section of the documentation already only applies to displays using the display buffer).
```suggestion
In each cycle, the display is automatically cleared before the lambda is executed. You can 
disable this behavior by setting ``auto_clear_enabled: false``.
```

Reply:
Done.